src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/ScanBlobScanLogHybridPollingStrategy.cs

```diff
@@ -223,10 +228,13 @@ private void ThrowIfDisposed()
             // record continuation token for next chunk retrieval
             containerScanInfo.ContinuationToken = blobSegment.ContinuationToken;
 
-            // if ending a cycle then copy currentSweepCycleStartTime to lastSweepCycleStartTime
-            if (blobSegment.ContinuationToken == null)
+            // if ending a cycle then copy currentSweepCycleStartTime to lastSweepCycleStartTime, if changed
+            if (blobSegment.ContinuationToken == null &&
+                containerScanInfo.CurrentSweepCycleLatestModified > containerScanInfo.LastSweepCycleLatestModified)
             {
                 containerScanInfo.LastSweepCycleLatestModified = containerScanInfo.CurrentSweepCycleLatestModified;
+                await _blobScanInfoManager.UpdateLatestScanAsync(container.ServiceClient.Credentials.AccountName,
+                    container.Name, containerScanInfo.LastSweepCycleLatestModified);
```

Comment:
The log scan goes back in time [2 hours](https://github.com/Azure/azure-webjobs-sdk/blob/master/src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/BlobLogListener.cs#L52). So if the container scan finds a blob modified at time T and persists a scan info T2 > T (e.g. another blob was modified a little after), if blob processing fails, container scan would never find that blob again. But presumably log scan will find it because it goes back 2 hours?